README.md

```diff
@@ -1616,8 +1628,15 @@ The ID token will be [verified and validated](https://github.com/okta/okta-auth-
 
 ```javascript
 authClient.token.parseFromUrl()
-.then(function(tokenOrTokens) {
+.then(function(res) {
```

Comment:
`res` implies an HTTP Response object, is there a better name?

Reply:
It is an object which contains a `tokens` hash and a `state` string.